tests/test_georaster_tiling.py

```diff
@@ -169,14 +145,17 @@ def test_get_tile_for_different_zoom_levels(self):
 
     def test_get_entire_all_raster(self):
         vr = self.small_read_only_virtual_geo_raster()
-        r = vr.get_tile(37108, 25248, 16, blocksize=None)
+        roi = GeoVector.from_xyz(37108, 25248, 16)
+        r = vr.crop(roi)
```

Comment:
in the future if we restrict `crop` to be in bounds only we should change this tests to use `get_window`